proxy/src/main/java/com/velocitypowered/proxy/command/builtin/VelocityCommand.java

```diff
@@ -174,10 +174,10 @@ public int run(final CommandContext<CommandSource> context) {
       if (version.getName().equals("Velocity")) {
         final TextComponent embellishment = Component.text()
             .append(Component.text()
-                .content("velocitypowered.com")
+                .content("papermc.io/software/velocity")
```

Comment:
This looks really meh

Reply:
yeah, any suggestions to make it better?

Reply:
Website?

Reply:
Homepage?

Reply:
At least old link still works and redirects to desired place

Reply:
if I were held at gunpoint, I'd probably just maybe name it PaperMC, anything else is just pointlessly verbose, imho